app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,8 @@
+class SessionsController < Devise::SessionsController
+  protected
+
+  def after_sign_in_path_for(user)
+     "/trucks"
+  end
+
+end
```

Comment:
Why is this being defined in these two different places? And why isn't it consistently being referenced as `trucks_path`, rather than the hard-coded version?